packages/core/src/lib/label.svelte

```diff
@@ -50,7 +50,7 @@ export { extraClasses as cx };
   )}
 >
   <span
-    class={cx('flex text-xs', {
```

Comment:
I agree with the removal, but I'm curious how this would do anything to the input slot, since it's a sibling rather than a child of this span.

I've definitely had this label render "incorrectly" horizontally on me, but every time it's been because I forgot to include `slot="input"` on my input element

Reply:
oh, i didn't realize there was an input slot 🙈 that did fix my issue. i'll close this PR then?